lib/msf/core/exploit/sqli.rb

```diff
@@ -0,0 +1,21 @@
+#
+# This mixin provides helpers to perform SQL injection
+# - provides a level of abstraction for common queries, for example, querying the table names
+# - implements blind and time-based SQL injection in a reusable manner
+# - Highly extendable (user can run any code to perform the requests, encode payloads and parse results)
+#
+module Msf::Exploit::SQLi
+  #
+  # This class represents any SQL injection object (an ancestor of any DBMS-specific SQL Injection class)
+  #
+  class SQLi
+
+  end
+
+  def initialize(info = {})
+    super
```

Comment:
This would be where @h00die had suggested registering the `sleepdelay`.

Reply:
@jmartin-r7  I thought about it, it's okay if I register sleepdelay even if SQLi is included for a non time-based injection? I don't see a way to know if the injection is time-based or not just at the time the module is included.

Reply:
What do you think about making it an `advanced` option?

https://github.com/rapid7/metasploit-framework/blob/76954957c740525cff2db5a60bcf936b4ee06c42/modules/exploits/windows/local/powershell_remoting.rb#L51-L54

@h00die what do you think?